### PR TITLE
Cleanup the heartbeat records based on the retention

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

File: manager/pkg/specsyncer/spec2db/controller/managedhub_spec_sync.go [hunk 1]
```diff
@@ -67,34 +64,6 @@ func (r *managedHubReconciler) Reconcile(ctx context.Context, request ctrl.Reque
 
 	// the managed hub is deleting then delete the data from database and then remove the finalizer
 	if !cluster.DeletionTimestamp.IsZero() {
-		db := database.GetGorm()
```
Comment (clyang82) [inline manager/pkg/specsyncer/spec2db/controller/managedhub_spec_sync.go:70]: Do not need to softly delete those resources?

Comment (yanmxa) [reply manager/pkg/specsyncer/spec2db/controller/managedhub_spec_sync.go:70]: No. We delete these resources on the [hub_management.go](https://github.com/stolostron/multicluster-global-hub/blob/main/manager/pkg/statussyncer/hubmanagement/hub_management.go)

**Pull request merged**